ietf-scitt-charter.md

```diff
@@ -21,8 +23,10 @@ Goals
 =====
 Based on an input document on the architecture (draft-birkholz-scitt-architecture-00), the WG will:
 
-1. Standardize the overall security flows for securing a software supply chain, covering the essential building blocks that make up the architecture, and
-2. specify these building blocks, employing the existing work already done within other IETF WGs such as COSE WG, and IETF RATS WG, as appropriate.
+1. Standardize the overall technical security flows for securing a software supply chain, including a focus on firmware, and covering the essential building blocks that make up the architecture, as well as
+2. specify these building blocks, employing the existing work already done within
```

Comment:
```suggestion
2. In addition to this, the WG shall employ the existing work already done within
```

Reply:
now the sentence started in 1. ends open ended in a "as well as". Do you intend the break up the numbered items into separate sentences?

Reply:
Ack